sinks/cortex/cortex.go

```diff
@@ -177,17 +177,26 @@ func (s *CortexMetricSink) Flush(ctx context.Context, metrics []samplers.InterMe
 	span, _ := trace.StartSpanFromContext(ctx, "")
 	defer span.ClientFinish(s.traceClient)
 	metricKeyTags := map[string]string{"sink": s.name, "sink_type": "cortex"}
-	emitPassOrFailSpan := func(err error, droppedMetrics int) {
-		if err == nil {
-			span.Add(ssf.Count(sinks.MetricKeyTotalMetricsFlushed, float32(len(metrics)), metricKeyTags))
-		} else {
-			emitMetricKeyTotalMetricsDropped(span, droppedMetrics, metricKeyTags)
+	sentMetrics := 0
```

Comment:
Prefer "flushedMetrics" to match the terminology w/ MetricKeyTotalMetricsFlushed